src/components/MovieCard/MovieCard.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import MovieCard from './MovieCard';
+
+describe('MovieCard', () => {
+    it('should render correctly', () => {
+        const mockData = {
```

Comment:
if you will use one mockData more than once it'll be nice to put it to the constants or mock